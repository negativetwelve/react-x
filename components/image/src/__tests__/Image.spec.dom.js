// Modules
import React from 'react';
import Image from '../Image';

/* eslint-disable no-undef */
describe('Image', () => {
  context('with no props', () => {
    itRenders(() => <Image />);
  });

  context('with style prop', () => {
    itRenders(() => <Image style={{padding: 12}} />);
  });
});
