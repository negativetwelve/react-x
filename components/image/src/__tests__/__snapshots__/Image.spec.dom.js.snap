// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image with no props renders snapshot 1`] = `
<div
  className="css-view-1dbjc4n r-flexBasis-1mlwlqe r-overflow-1udh08x r-zIndex-417010"
>
  <div
    className="css-view-1dbjc4n r-backgroundColor-1niwhzg r-backgroundPosition-vvn4in r-backgroundRepeat-u6sd8q r-backgroundSize-4gszlv r-bottom-1p0dtai r-height-1pi2tsx r-left-1d2f490 r-position-u8s1d r-right-zchlnj r-top-ipm5af r-width-13qz1uu r-zIndex-1wyyakw"
  />
</div>
`;

exports[`Image with style prop renders snapshot 1`] = `
<div
  className="css-view-1dbjc4n r-flexBasis-1mlwlqe r-overflow-1udh08x r-zIndex-417010"
  style={
    Object {
      "paddingBottom": "12px",
      "paddingLeft": "12px",
      "paddingRight": "12px",
      "paddingTop": "12px",
    }
  }
>
  <div
    className="css-view-1dbjc4n r-backgroundColor-1niwhzg r-backgroundPosition-vvn4in r-backgroundRepeat-u6sd8q r-backgroundSize-4gszlv r-bottom-1p0dtai r-height-1pi2tsx r-left-1d2f490 r-position-u8s1d r-right-zchlnj r-top-ipm5af r-width-13qz1uu r-zIndex-1wyyakw"
  />
</div>
`;
